README.md

```diff
@@ -1,2 +1,198 @@
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [📝 License ](#-license-)
+
+# SCHOOL LIBRARY APP
+A simple project written in Ruby that uses Object Oriented Programming to record what books are in the library and who borrows them. The app will allow you to:
+
+Add new students or teachers.
+Add new books.
+Save records of who borrowed a given book and when.
+
+## Getting Started
+
+**Please follow the steps below to setup this project in your local computer
+<a name="readme-top"></a>
+
+### How to build the "SCHOOL LIBRARY APP" 
+
+### Prerequisites
+A basic understanding of Ruby and Git
+### Usage
+1. Install Ruby
+2. Clone this repo `git clone https://github.com/Bettenoch/OOP-School-Library.git`
+
+
+## 🛠 Walk through <a name="walk-through"></a>
+
+<ul>
+  <li><a href="https://github.com/microverseinc/curriculum-ruby/blob/main/oop/sneak_peek.md">See the Project Requirement</a></li>
+  <li>If required you should personalize the content of your page. Choose a topic that is different than the one in the original design</li>
+
+
+
+## 🛠 Built With <a name="built-with"></a>
+
+**_feature-One_**
+
+- Implementation of ruby classes and methods.
+
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <ul>
+    <li><a href="https://www.w3.org/html/">RUBY</a></li>
+     <li><a href="https://www.w3.org/html/">Rubocop</a></li>
+  </ul>
+</details>
+
+
+<!-- Features -->
+
+### Tools <a name="tools"></a>
+
+
+  <details>
+    <summary>Version Control, CI/CD, Hosting Service</summary>
+      <ul>
+        <li><a href="https://pages.github.com/">Github Pages</a></li>
+        <li><a href="https://github.com/features/actions">Github Actions</a></li>
+        <li><a href="https://git-scm.com/">Git</a></li>
+      </ul>
+  </details>
+  <details>
+    <summary>IDE, Desktop Apps, Other Tools</summary>
+      <ul>
+        <li><a href="https://code.visualstudio.com/">Visual Studio Code</a></li>
+        <li><a href="https://desktop.github.com/">Github Desktop</a></li>
+        <li><a href="https://www.behance.net/">Behance</a></li>
+      </ul>
+  </details>
+
+### Key Features <a name="key-features"></a>
+
+
+DBMS
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+- git version 2.38.x
+- node.js version > 12.x
+- IDE
+- browser (chrome, firefox, edge, safari)
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```sh
+  cd OOP-school=library
+
+  git https://github.com/Bettenoch/OOP-School-Library.git
+ 
+```
+
+### Install
+
+Install this project with:
+
+```sh
+  cd OOP-school=library
+  node npm install
+```
+
+### Usage
+
+To run the project, execute the following command:
+run live server
+
+### Run tests
+
+To run tests, run the following command:
+Run Github Actions Test
+
+```sh
+  npm test
```

Comment:
- [ ] Hey there! 🚀 Do you need to use npm test to run tests for Ruby projects? Let's dive in and clarify.

- In general, Ruby projects use testing frameworks like RSpec or MiniTest, which are specifically designed for Ruby code. To run these tests, you typically use commands like rspec or rake test, depending on the testing framework you're using.

- On the other hand, npm test is a command used in JavaScript and Node.js projects to run tests written in JavaScript, often using frameworks like Jest or Mocha. It's not directly related to Ruby testing.

- So, in summary, for Ruby tests, you would use Ruby-specific testing commands, not npm test. 🧪💎🔍